### ISSUE TITLE
Implement BoxManager Component

### ISSUE THREAD

**Issue by philipmjohnson** [openpowerquality/opq#67]
Please do work for this task in a branch called issue-67.

A user must be able to manage an OPQ Box. This involves:

 * View existing boxes tied to their account
 * Register new boxes to their account
 * Edit Box label (so user doesn't have to keep track of serial numbers)
 * Edit Box location (zip code, so we have an approximate location on each device, while still providing a degree of privacy)
 * Edit Device permissions
 * View basic device statistics (device uptime, most recent heartbeat)
  * Manage its location
  * Manage its nickname
  * Manage whether it is online or offline
  * Manage whether its data is public or private.  Public means the data appears on the landing page and in all other user's pages.  Private means only owners can see the box and/or its data, and only after logging in.

**Comment by philipmjohnson** [comment]
I'm going to close this for now.  We have basic box and user management via the settings file.  Use of MAC address authentication is deferred for this milestone.
